managed/src/main/java/com/yugabyte/yw/commissioner/tasks/ResumeUniverse.java

```diff
@@ -55,18 +55,18 @@ public void run() {
       Set<NodeDetails> tserverNodes = new HashSet<>(universe.getTServers());
       Set<NodeDetails> masterNodes = new HashSet<>(universe.getMasters());
 
+      createStartMasterTasks(masterNodes).setSubTaskGroupType(
+          SubTaskGroupType.StartingNodeProcesses);
       for (NodeDetails node : tserverNodes) {
         createTServerTaskForNode(node, "start").setSubTaskGroupType(
             SubTaskGroupType.StartingNodeProcesses);
       }
-      createStartMasterTasks(masterNodes).setSubTaskGroupType(
-          SubTaskGroupType.StartingNodeProcesses);
-
-      createWaitForServersTasks(tserverNodes, ServerType.TSERVER)
-          .setSubTaskGroupType(SubTaskGroupType.ConfigureUniverse);
+      
       createWaitForServersTasks(masterNodes, ServerType.MASTER)
           .setSubTaskGroupType(SubTaskGroupType.ConfigureUniverse);
-
+      createWaitForServersTasks(tserverNodes, ServerType.TSERVER)
```

Comment:
I'd actually recommend doing this after starting the service. So basically, start masters, wait for them to be up and then start the tservers.

Reply:
Ok, so that means, start master, wait, start t server and wait for them to up, right?